reV/hybrids/hybrids.py

```diff
@@ -0,0 +1,1170 @@
+        correctly and exist in the input data. It also verifies that
+        the `allowed_ratio` is correctly formatted.
+
+        """
+        self._validate_limits_cols_prefixed()
+        self._validate_fillna_cols_prefixed()
+        self._validate_ratio_cols_type()
+        self._validate_ratio_cols_prefixed()
+        self._validate_ratio_cols_exist()
+        self._validate_ratio_cols_fixed_input()
+        self._validate_ratio()
+
+    def _validate_limits_cols_prefixed(self):
+        """Ensure the limits columns are formatted correctly.
+
+        This check is important because the limiting happens
+        after the meta has been merged (so columns are already prefixed),
+        but before the hybrid columns are computed. As a result, the limits
+        columns _must_ have a valid prefix.
+
+        Raises
+        ------
+        InputError
+            If limits columns are not prefixed correctly.
+        """
+        for col in self._limits:
+            self.__validate_col_prefix(
+                col, (SOLAR_PREFIX, WIND_PREFIX), input_name='limits'
+            )
+
+    @staticmethod
+    def __validate_col_prefix(col, prefixes, input_name):
+        """Validate the the col starts with the correct prefix. """
+
+        missing = [not col.startswith(p) for p in prefixes]
+        if all(missing):
+            msg = ("Input {0} column {1!r} does not start with a valid "
+                   "prefix: {2!r}. Please ensure that the {0} column "
+                   "names specify the correct resource prefix.")
+            e = msg.format(input_name, col, prefixes)
+            logger.error(e)
+            raise InputError(e)
+
+    def _validate_fillna_cols_prefixed(self):
+        """Ensure the fillna columns are formatted correctly.
+
+        This check is important because the fillna step happens
+        after the meta has been merged (so columns are already prefixed),
+        but before the hybrid columns are computed. As a result, the fillna
+        columns _must_ have a valid prefix.
+
+        Raises
+        ------
+        InputError
+            If fillna columns are not prefixed correctly.
+        """
+        for col in self._fillna:
+            self.__validate_col_prefix(
+                col, (SOLAR_PREFIX, WIND_PREFIX), input_name='fillna'
+            )
+
+    def _validate_ratio_cols_type(self):
+        """Ensure that the ratio column keys are `RatioColumns` instances.
+
+        Raises
+        ------
+        InputError
+            If ratio_cols is not an instance of `RatioColumns`.
+        """
+
+        for cols in self._ratios:
+            if not isinstance(cols, RatioColumns):
+                msg = ("Key {!r} for input 'ratios' not understood. "
+                       "Please make sure this value is an instance of "
+                       "`RatioColumns`")
+                e = msg.format(cols)
+                logger.error(e)
+                raise InputError(e) from None
```

Comment:
I dont think you need the `from None` here? I thought you just used `raise X from E` when you're catching a different exception E